src/v2/components/Preferences.jsx

```diff
@@ -0,0 +1,176 @@
+// -----------------------------------------------------------------------------
+// Copyright 2022 Open Climate Tech Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+import React, {useEffect, useRef} from 'react'
+
+import Duration from '../modules/Duration.mjs'
+import query from '../modules/query.mjs'
+
+const Props = {
+  CENTER: [34.3987, -119.8199], // Santa Barabara, CA.
+  CAMERA: {
+    color: '#c00',
+    fill: '#c00',
+    fillOpacity: 0.73,
+    opacity: 0.93,
+    radius: 3,
+    weight: 1
+  },
+  CAMERA_RADIUS: {
+    color: '#c00',
+    fill: '#c00',
+    fillOpacity: 0.07,
+    opacity: 0.13,
+    radius: 1.50,
+    weight: 1.50
+  },
+  MAP: {
+    zoomAnimationThreshold: 18,
+    zoomControl: true
+  },
+  TILE_LAYER: {
+    attribution: '© <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a>',
+    maxZoom: 18
+  },
+  SET_VIEW: {
+    animate: true,
+    duration: Duration.SECOND / 1000
+  },
+  URL_TEMPLATE: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'
+}
+
+const DegreesOffet = {
+  // 20 miles estimated coverage / 54.6 miles per degree of longitude.
+  // 20 miles estimated coverage / 69 miles per degree of latitude.
+  EAST: 20 / 54.6,
+  NORTH: 20 / 69,
+  SOUTH: -20 / 69,
+  WEST: -20 / 54.6
+}
+
+/**
+ * @returns {React.Element}
+ */
+export default function Preferences(props) {
+  const stateRef = useRef({})
+
+  useEffect(() => {
+    const {L} = window
+    const ref = initializeMap(stateRef.current)
+    const {map} = ref
+
+    query.get('/api/monitoredCameras').then(({data}) => {
+      if (ref.points != null) {
+        ref.points.forEach((x) => x.remove())
+      }
+
+      map.on({
+        zoom: () => {
+          const {markers, points} = ref
+
+          if (points != null && points.length > 0) {
+            if (markers != null) {
+              markers.forEach((x) => x.remove())
+            }
+
+            const radius = calculateRadius(map, points[0].getLatLng())
+            ref.markers = points.map((x) => {
+              const marker = L.circleMarker(x.getLatLng(), {...Props.CAMERA_RADIUS, radius}).addTo(map)
+              x.remove().addTo(map)
+              return marker
+            })
+          }
+        }
+      })
+
+      ref.points = data.map(({Latitude: lat, Longitude: lng}) => L.circleMarker([lat, lng], Props.CAMERA).addTo(map))
```

Comment:
My bad on not catching upper (sqlite) vs. lower (postgres) case on 'Latitude' and 'Longitude' .  Please apply PR https://github.com/open-climate-tech/checkfire/pull/39 and adjust the code to use lowercase

Reply:
Okay

Reply:
Done